nvim/init.lua

```diff
@@ -557,6 +759,59 @@ require('lazy').setup({
     end
   },
 
+  -- auto save
+  {
+    "okuuva/auto-save.nvim",
+    cmd = "ASToggle", -- optional for lazy loading on command
+    event = { "InsertLeave", "TextChanged" }, -- optional for lazy loading on trigger events
+    opts = {
+      execution_message = {
+        enabled = false,
+      },
+      debounce_delay = 5000,
+    },
+  },
+
+  {
+    "saccarosium/nvim-whitespaces",
+    lazy = false,
+    opts = {
+      handlers = {},
+    },
+  },
+
+  {
+    "booperlv/nvim-gomove",
+    event = "VeryLazy",
+    opts = {
+      handlers = {},
+    },
+  },
+
+  -- Docker
+  -- LazyDocker app is required https://github.com/mgierada/lazydocker.nvim?tab=readme-ov-file#-installation
+  {
+    "mgierada/lazydocker.nvim",
+    event = "VeryLazy",
```

Comment:
Plugins for docker && DB should definitely be included in the base version. Thank you for the suggestions.